third_party/wabt/src/interp/jit/binary-reader-jit.cc

```diff
@@ -433,18 +434,40 @@ Result BinaryReaderJIT::OnBinaryExpr(Opcode opcode) {
     case Opcode::I64ShrU:
       result = LocationInfo::kEightByteSize;
       break;
+    case Opcode::F32Add:
+    case Opcode::F32Sub:
+    case Opcode::F32Mul:
+    case Opcode::F32Div:
+    case Opcode::F32Max:
+    case Opcode::F32Min:
+    case Opcode::F32Copysign:
+      group = Instruction::BinaryFloat;
+      result |= LocationInfo::kFloat;
```

Comment:
I think you can use `=` here.

Reply:
How is this resolved? It is still `|=`. Below `result = LocationInfo::kEightByteSize | LocationInfo::kFloat;` is used, you can also use `result = LocationInfo::kFourByteSize | LocationInfo::kFloat;` here. The advantage is that the previous value does not need to be preserved by the compiler.